src/coreclr/jit/emitarm64.cpp

```diff
@@ -5633,9 +5631,7 @@ void emitter::emitIns_R_R_I(instruction     ins,
             break;
 
         case INS_ldurh:
-        case INS_ldapurh:
         case INS_sturh:
```

Comment:
While you're here, could you please move these cases up to `INS_ldurb/INS_sturb` since they have the same behavior?

Reply:
Addressed. They indeed have the same issue but we never use them 🙂